### PR TITLE
Feature/174 add a route to implement `containers/:containerId`

### LINKED ISSUE #174: add a route to implement `/containers/{containerId}`
## Feature Title

Implement [this endpoint](https://genesis-tech-tribe.github.io/nishiki-documents/web-api/#tag/container/paths/~1containers~1%7BcontainerId%7D/get)
add containers POST endpoint for getting a container Info

## Acceptance Criteria

- [ ] Criterion 1
- [ ] Criterion 2

## Technical Details

- API Changes:
- Database Changes:
- Dependencies:

## Additional Context

Any other internal context or references.

### PR DESCRIPTION
**Pull request by taniguchiiqqq**
## Overviews of implementation
add a route to implement `containers/:containerId`
add containerWithGroupDtoMapper to add Group info to ContainerDTO

## Review points
Is the containerWithGroupDtoMapper implementation appropriate?

### CODE REVIEW

**Comment by hitohata** [comment]
@taniguchiiqqq 
Could you check my revised one? If it's no problem, I'll merge this PR.

File: backend/main/test/Group/Query/FindContainerQuery.test.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+const USER_ID = UserId.generate();
+const GROUP_ID = "4898aa6e-ef03-4ae2-86c4-1db39b25a0bc";
+const GROUP_NAME = "group name";
+const CONTAINER_ID = "6112947e-e2bf-4a68-b3e5-9c39a937a297";
+const CONTAINER_NAME = "container name";
+
+describe("find container use case", () => {
+	let mockNishikiDynamoClient: NishikiDynamoDBClient;
+	let useCase: FindContainerQuery;
```
Comment (taniguchiiqqq) [inline backend/main/test/Group/Query/FindContainerQuery.test.ts:15]: Could you change this variable name to `query`? But it's not a big deal, so it's up to you.

**Comment by taniguchiiqqq** [comment]
@hitohata 
Thanks a lot!! Apart from the comment, it looks so good.

File: backend/main/test/Group/Query/FindContainerQuery.test.ts (hunk 1 above)
Comment (hitohata) [reply backend/main/test/Group/Query/FindContainerQuery.test.ts:15]: Thank you for pointing it out. I fixed it.

**Review by hitohata — APPROVED** [review]
This PR has been also checked by @taniguchiiqqq.

**Pull request merged**